### DISCUSSION TITLE
Feedback for “Artifacts - Generative UI”

### DISCUSSION THREAD

**Discussion by stanyslassz** [LibreChat-AI/docs#179] [Q&A]
Hello Dany and thank you for the work

Artifacts and fantastic.

Is there a way to download the file generated pack? Because it seems to have dependencies and just copying the code is not relevant, and often asks for the creation of the structure.

**Comment by danny-avila** [comment]
Thank you! I'm not sure what you mean by the file generated pack, can you provide some screenshots maybe detailing the request?
